.github/workflows/bank-sdk.publish.docs.yml

```diff
@@ -11,6 +11,10 @@ on:
         required: true
   workflow_dispatch:
 
+concurrency:
```

Comment:
[Suggestion]: @GeraltRiv it's a part of release workflow, let's skip concurrency for `*.publish.docs.yml`  files

Reply:
Fixed

Reply:
Fixed